### ISSUE TITLE
Deleting Service Account returns 500 Internal Server Error

### ISSUE THREAD

**Issue by evermediocre** [featurehub-io/featurehub#448]
**Describe the bug**
Deleting a service account shows a dialog with this error:

**Couldn't delete service account "Service Account 1"**
```
TypeError: null: type 'minified:va' is not a subtype of type 'bool' Stack trace: TypeError: null: type 'minified:va' is not a subtype of type 'bool'
    at Object.b (http://localhost:8085/main.dart-1623048746.js:3872:3)
    at Object.aTt (http://localhost:8085/main.dart-1623048746.js:4508:27)
    at jP.b6z [as a] (http://localhost:8085/main.dart-1623048746.js:4503:3)
    at SC.cO (http://localhost:8085/main.dart-1623048746.js:38063:22)
    at Object.o (http://localhost:8085/main.dart-1623048746.js:5544:19)
    at http://localhost:8085/main.dart-1623048746.js:46953:17
    at aGC.a (http://localhost:8085/main.dart-1623048746.js:5554:71)
    at aGC.$2 (http://localhost:8085/main.dart-1623048746.js:38080:23)
    at aFq.$1 (http://localhost:8085/main.dart-1623048746.js:38072:30)
    at XX.r5 (http://localhost:8085/main.dart-1623048746.js:39375:41)
```
Http Request Info:
```
Request URL: http://localhost:8085/mr-api/service-account/7eb76df9-7344-4b9c-9779-47c769289057
Request Method: DELETE
Status Code: 500 Internal Server Error
Remote Address: [::1]:8085
Referrer Policy: strict-origin-when-cross-origin
```

**Which area does this issue belong to?**

* [x] FeatureHub Admin Web app

**To Reproduce**
1. Go to 'Portfolio Settings -> Service Accounts'
2. Delete a service account
3. A dialog will appear showing the error above.

**Desktop (please complete the following information):**
 - OS: Windows
 - Platform: Docker for Windows
 - Browser: any
 - Version: latest

**Comment by IrinaSouth** [comment]
Hi there, thanks for reporting the bug. I managed to reproduce - it turns out we have a restriction on the Service account name length in our backend :) Should be fixed soon

**Comment by rvowles** [comment]
This is because it archives rather than deletes the record, and to ensure uniqueness in the name it was adding a timestamp which took up too much space.

**Comment by rvowles** [comment]
will be released in 1.3.4 shortly.
